uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -70,10 +68,10 @@ define([
           var begin = (this.currentPage - 1) * this.numPerPage,
             end = begin + this.numPerPage;
 
-          this.filteredbreakdown = vm.balance.change.breakdown.slice(begin, end);
+          this.filteredbreakdown = parentThis.balance.change.breakdown.slice(begin, end);
```

Comment:
it will not work as the bind will be for parent context and not for object context